functional_tests/testdata_histogram/expected/v1.29/controller_manager_metrics.yaml

```diff
@@ -4883,4 +4883,4 @@ resourceMetrics:
             name: node_ipam_controller_cidrset_allocation_tries_per_request
         scope:
           name: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver
-          version: v0.109.0
+          version: v0.110.0
```

Comment:
https://github.com/signalfx/splunk-otel-collector-chart/pull/1482 should make it so you don't have to updates these [controller_manager_metrics.yaml](https://github.com/signalfx/splunk-otel-collector-chart/pull/1481/files#diff-51ebe0bb5fea8cc6a7fda3c168b0c4eb19bc5f63625f5207cd88767309aa3116) and [coredns_metrics.yaml](https://github.com/signalfx/splunk-otel-collector-chart/pull/1481/files#diff-8a6dcdfda9b9238fbb81a49821b5932fc9b0398d7f2938f47e39982b2cd30e08) tests

Reply:
OK, got it, I added it initially as pipeline was failing here 
I removed that change from PR

Reply:
Thank you